lib/server.js

```diff
@@ -718,7 +718,7 @@ WalletService.prototype._getUtxosForAddresses = function(addresses, cb) {
       var u = _.pick(utxo, ['txid', 'vout', 'address', 'scriptPubKey', 'amount', 'satoshis', 'confirmations']);
       u.confirmations = u.confirmations || 0;
       u.locked = false;
-      u.satoshis = u.satoshis ? +u.satoshis : Utils.strip(u.amount * 1e8);
+      u.satoshis = _.isNumber(u.satoshis) ? +u.satoshis : Utils.strip(u.amount * 1e8);
```

Comment:
I did at first but people will keep writing _.range(x). I'm sure since this is just the test environment, it's not a big deal. And yes, your fix makes it more robust no doubt.